seatunnel-api/src/main/java/org/apache/seatunnel/api/configuration/ReadonlyConfig.java

```diff
@@ -144,6 +144,10 @@ private Object getValue(String key) {
         }
     }
 
+    public Map<String, Object> copyData() {
```

Comment:
Look likes this method not a deep copy. It will cause bug.

Reply:
removed this method